app/assets/stylesheets/custom.scss

```diff
@@ -68,7 +68,7 @@ ul {
   word-spacing: 3px;
 }
 
-.about-question {
+.about--question {
```

Comment:
I'm preserving `about--question` because it seems appropriate. I did add an extra dash for consistency with BEM

Reply:
`--` has different meaning than `__` in bem. I just added a commit to fix this.